.circleci/config.yml

```diff
@@ -101,6 +169,54 @@ jobs:
               exit 1
             fi
 
+  ## ------------------ Check python code against known security issues ------------------
+  cve-repo-scan:
+    parameters:
+      with_merge: # Part of the PR-merge workflow
+        type: boolean
+        default: false
+    docker:
+      # https://circleci.com/developer/images/image/cimg/python
+      - image: cimg/base:current
+    steps:
+      - when:
+          condition: << parameters.with_merge >>
+          steps:
+            - disable_build_if_not_pr
+      - checkout
+      - when:
+          condition: << parameters.with_merge >>
+          steps:
+            - merge-pr-branch
+      - sm-build/trivy_scan_filesystem:
+          report_json_name: fs_report.json
+          report_html_name: fs_report.html
+      - store_artifacts:
+          name: Save Filesystem Report
+          path: fs_report.html
+
+  ## ------------- Check python code and container image against known security issues -----------
+  cve-image-scan:
+    docker:
+      # https://circleci.com/developer/images/image/cimg/python
+      - image: cimg/base:current
+    steps:
+      # Checkout the code from the master branch
+      - checkout
+      # Perform the security scan on the filesystem
+      - sm-build/trivy_scan_filesystem:
+          report_json_name: fs_report_master.json
+          report_html_name: fs_report_master.html
+      # Store the artifacts for the master branch scan
+      - store_artifacts:
+          name: Save Master Branch Filesystem Report
+          path: fs_report_master.html
+      # The following will mark the build as failed if there are issues in the report.
+      - sm-build/trivy_notify_on_issues:
+          slack_channel: monitoring-sdk
```

Comment:
The only question I have is should this be a separate channel, or should be piggy back on the MA Monitoring channel?